apps/aeutil/lib/serialization.ex

```diff
@@ -129,25 +130,25 @@ defmodule Aeutil.Serialization do
   def serialize_value(value, type) when is_binary(value) do
     case type do
       :prev_hash ->
-        Header.bech32_encode(value)
+        Header.base58_encode(value)
 
       :txs_hash ->
-        SignedTx.bech32_encode_root(value)
+        SignedTx.base58_encode_root(value)
 
-      :chain_state_hash ->
-        ChainState.bech32_encode(value)
+      :root_hash ->
+        ChainState.base58_encode(value)
 
       :from_acc ->
-        Aewallet.Encoding.encode(value, :ae)
+        Account.base58_encode(Kernel.to_string(value))
 
       :to_acc ->
-        Aewallet.Encoding.encode(value, :ae)
+        Account.base58_encode(Kernel.to_string(value))
 
       :signature ->
         base64_binary(value, :serialize)
 
       :proof ->
-        Aewallet.Encoding.encode(value, :ae)
+        Account.base58_encode(value)
```

Comment:
why does the proof use the this?

Reply:
Because this is how the proof was being encoded [before](https://github.com/aeternity/elixir-node/pull/263/files#diff-92c5a5701af6ae4117cba6975aba699aL88)

Reply:
then I didn't catch it earlier, what is the reasoning for this, do we need a specific encoding for this at all? [integer()] shouldn't, right?